internal/cli/err/errors.go

```diff
@@ -1,34 +1,34 @@
-package cli
+package clierr
```

Comment:
I know this is a common pattern in the `baas` repo, but it becomes hard to follow since the package normally is the name of the package. Any reason we can't have this in the `cli` package itself?

Reply:
it couldn't be in the `cli` package due to import cycles... but I can move things around if this pattern is critical to remove.  but basically I wanted `Config.Load() error` to be able to return a `cli.Error`, and by forcing `Feedback` methods to be "error-free" (see comment above), the `ui` package wants access to `clieerr.HandleErr` to catch anything issues there

Reply:
CLI errors have been moved back to the `cli` package :)